base/interactiveutil.jl

```diff
@@ -92,13 +92,13 @@ edit(file, line::Integer) = error("could not find source file for function")
 
 if Sys.iswindows()
     function less(file::AbstractString, line::Integer)
-        pager = get(ENV, "PAGER", "more")
+        pager = shell_split(get(ENV, "PAGER", "more"))
         g = pager == "more" ? "" : "g"
```

Comment:
Don't you want to look at `pager[1]` here? Since `pager` will be an array here this test will always fail (silently).